crates/proto-common/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "seda-proto-common"
+version = "0.1.0"
```

Comment:
What we did for the TS variant is to have the package version exactly the same as the chain version from which the proto files were taken. I think that helps making things a little clearer when protobuf breaks down between components.

Reply:
What if it's not tagged a new version yet?

Reply:
I coordinated with HY on what the next version should be and changed it to that :).